module4/projects/solo_stretch_tech/index.md

```diff
@@ -3,14 +3,137 @@ title: Solo Stretch Tech
+### All deliverables will be turned in via **[this google sheet](https://docs.google.com/spreadsheets/d/1dkhqQctc_6nylYyQII22zLU0He_P1k_xfaJOFngV7dM/edit?usp=sharing)**.
+</section>
+
+<section class="dropdown">
+### Day 2
+- Project Proposal: Make a copy of [this document](https://docs.google.com/document/d/1xUWFfFzSJ09GlWi-55942H2RgDJ8GZN-dTMhNsgakig/edit?usp=sharing) and fill it out. Add the link to your proposal to the google sheet. An instructor will review your proposal. You should not move forward until your proposal has been approved. 
+- Stretch Techs: List the stretch techs you have chosen for your project. 
+</section>
+
+<section class="dropdown">
+### Day 4
+
+- FE Design: We expect to see a thought-out design for your frontend before you start coding. Use any tool you'd like (Canva, Figma, Excalidraw, etc) to create a solid wireframe for your app. If you struggle with design, keep it simple. Less is more when it comes to FE design! Look at similar apps out there and take note of what you like about them. Use those features as inspiration.
+- BE Planning Docs: We also expect to see a strong schema design before you start coding. Use any tool you'd like for this. We want to see what data you plan to manage, how it will be organized, and what the relationships between the data will be. 
+- Stretch Tech Learning Resources: Please find at least 2 resources per stretch tech (4 total) that you will use for learning. This could be the official docs, a tutorial, a YouTube series, etc. 
+</section>
+
+<section class="dropdown">
+### Day 7
+
+- Repo(s): Please create your repo(s) and link them in the google sheet. You don't need to have any code in there yet.
+- Project Board: You will link your project board, which should include:
+  - At least 4 columns, i.e. `To Do`, `In Progress`, `In Review`, `Merged/Done`
+  - At least 6 tickets, with each ticket including a user story
+  - All tickets should be converted to **[issues](https://docs.github.com/en/issues)** in your repo
+      - Issues are sometimes disabled. To enable them go to Settings in the upper right corner, then scroll down to the Features section. Check the Issues box to enable them.
+  - (optional) The use of **[labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)** could be very helpful in organizing your Project Board
+    - To view labels on the project board, click the down carrot on the View tab (top left of page) > Fields > ✔️ Labels.
+</section>
+
+<section class="dropdown">
+### Day 9
+
+- First PR: We'd love to (finally) see some code! Choose any ticket from your project board and get it done! Push up a PR and drop the link in the google sheet. This can be BE or FE code. Your PR should [close the associated ticket by referencing the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue). You can merge this PR anytime, we will not be doing a code review.
```

Comment:
I might just be way too obtuse, but I read this as students both start the PR and submit/merge all on Day 9. I'd love a normal person's take on if the read it the same way or if we expect students will spend both days 8 and 9 on this. If we think there will be confusion, it might be worth clarifying that Day 8 should be the start of coding.

Reply:
And maybe also a reminder in this Day 9 dropdown that if they cannot finish the issue between days 8-9, the issue is too large and should be broken out into smaller tasks.

Reply:
I can add a note to the end of Day 7 and say "It's time to start coding!"